components/dropdown/README.md

```diff
@@ -1,10 +1,12 @@
+
 # Dropdowns
-## Dropdown Openers
+A Dropdown is a button that opens a floating container to offer menu items or other content.
 
-### d2l-dropdown
+## Opener: Dropdown [d2l-dropdown]
 
 `d2l-dropdown` is a generic opener for dropdown content (`d2l-dropdown-content`, `d2l-dropdown-menu` or `d2l-dropdown-tabs`) enabling alternate opener implementation using existing elements/components. Provide and indicate your own opener element with the class attribute value `d2l-dropdown-opener`.  Wire-up is automatic.
 
+<!-- docs: demo live name:d2l-dropdown autoSize:false align:flex-start size:medium -->
```

Comment:
The demos are currently static sized (`medium`) but the dropdown's start closed. @margaree To match what you did with date/time do you think that these should begin open (or maybe just the ones dealing with dropdown content)?

Reply:
Check with Jeff to be sure but I think we'd likely want to have them open as well and adopt a similar strategy as that page for how `opened` gets handled.